rhp/v3/rpc_test.go

```diff
@@ -150,6 +153,17 @@ func TestAppendSector(t *testing.T) {
 			t.Fatal("downloaded sector doesn't match")
 		}
 	}
+
+	// assert ReadSector exposes ErrSectorNotFound
```

Comment:
Is there no `ReadSector` test we can extend rather than `TestAppendSector`?

Reply:
Didn't want to add one just for this small regression test 🤷‍♂️ . I don't mind extracting it into a separate test though, Nate be the judge?